src/main/js/helpers/axis.js

```diff
@@ -997,20 +998,35 @@ const determineOutlierStretchFactor = (config) => {
     };
 };
 
+/**
+ * Checks if the Axis label needs to be truncated and returns the truncated value
+ *
+ * @private
+ * @param {string} label - Axis label display property
+ * @param {number} charLimit  - label character limit on axis
+ * @returns {string} if more than constants.DEFAULT_LABEL_CHARACTER_LIMIT then truncates,
+ * normal label otherwise
+ */
+const formatTrackLabel = (label, charLimit) =>
```

Comment:
Are we using this function for all graphs? Then should be updated to `formatLabel`?